### PR TITLE
Many QoL updates

### PR DESCRIPTION
**Pull request by jp00p**
Updated profile terminology in the database and in the code: Profile cards became profile photos, profile badges became profile stickers.

Made the bot send an announcement to the admin channel when a new person posts an intro

Centered badges in their slots

Updated level up messaging

Update shop items

Tweaked wordcloud some more

Other tiny things

### CODE REVIEW

File: data/seed-db.sql [hunk 1]
```diff
@@ -17,10 +17,11 @@ CREATE TABLE IF NOT EXISTS users (
   spins int(11) NOT NULL DEFAULT 0,
   jackpots int(11) NOT NULL DEFAULT 0,
   wager int(11) NOT NULL DEFAULT 1,
-  profile_card longtext DEFAULT NULL,
-  profile_badge varchar(255) DEFAULT NULL,
+  profile_photo longtext DEFAULT NULL,
+  profile_sticker_1 varchar(255) DEFAULT NULL,
+  profile_sticker_2 varchar(255) DEFAULT NULL,
+  profile_sticker_3 varchar(255) DEFAULT NULL,
```
Comment (zmattingly) [inline data/seed-db.sql:23]: See earlier comment regarding a join table here?

**Review by zmattingly — COMMENTED** [review]
Looks gewd, just a few comments and one mainly about potentially using a join table for the sticker slots.

File: README.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ Bot commands are triggered by typing an exclamation point followed by a command.
...
 | `!qget [user]`                                                                                                                                    | [q.py](commands/q.py)                         | Get the information in mysql for a specific user                                                                                        |
-| `!qset [user] [score \| spins \| jackpots \| wager \| high_roller \| chips \| profile_card \| profile_badge \| xp] [new-value]`                   | [q.py](commands/q.py)                         | Set a value of a specific user in mysql                                                                                                 |
+| `!qset [user] [score \| spins \| jackpots \| wager \| high_roller \| profile_photo \| profile_sticker_1 \| xp] [new-value]`                   | [q.py](commands/q.py)                         | Set a value of a specific user in mysql                                                                                                 |
```
[the remark this answers is not in the document]
Comment (mathew-fleisch) [reply README.md:113]: Protip: turn wordwrap off

File: commands/profile.py [hunk 3]
```diff
@@ -235,10 +235,10 @@ async def profile(ctx:discord.ApplicationContext, public:str):
   base_bg.paste(padd_frame, (0, 0), padd_frame)
 
   # put sticker on
-  if user["profile_badge"]:
+  if user["profile_sticker_1"]:
     sticker_image = Image.open("./images/profiles/template_pieces/lcars/sticker-1.png").convert("RGBA")
     sticker_mask  = Image.open("./images/profiles/template_pieces/lcars/sticker-1-mask.png").convert("L").resize(sticker_image.size)
-    sticker       = Image.open(f"./images/profiles/badges/{user['profile_badge']}").convert("RGBA").resize(sticker_image.size)
+    sticker       = Image.open(f"./images/profiles/stickers/{user['profile_sticker_1']}").convert("RGBA").resize(sticker_image.size)
```
[the remark this answers is not in the document]
Comment (mathew-fleisch) [reply commands/profile.py:241]: + timestamp

File: data/seed-db.sql (hunk 1 above)
Comment (mathew-fleisch) [reply data/seed-db.sql:23]: agree. not very scalable 🤔

File: commands/agimus.py [hunk 4]
```diff
@@ -66,6 +71,7 @@ def handle_special_questions(question):
   special_questions = {
     "are you a friend of desoto": "Best boss I ever had!",
     "are you a FoD": "Best boss I ever hand!",
+    "are you an FoD": "Best boss I ever hand!",
```
[the remark this answers is not in the document]
Comment (jp00p) [reply commands/agimus.py]: fixed!

File: commands/agimus.py [hunk 5]
```diff
@@ -79,7 +85,7 @@ def handle_special_questions(question):
   return completion_text
 
 def handle_openai_query(question):
-  prompt_start = "You are a mischievous computer intelligence named AGIMUS. You are a bot on a Discord server called The USS Hood for fans of The Greatest Generation and The Greatest Discovery Star Trek Podcasts, hosted by Ben Ahr Harrison and Adam Pranica. The Fans are called The Friends of DeSoto. You cannot self-destruct the ship. Answer the following prompt"
+  prompt_start = "You are a mischievous computer intelligence named AGIMUS. You are a bot on a Discord server called The USS Hood for fans of The Greatest Generation and The Greatest Discovery Star Trek podcasts, hosted by Ben Ahr Harrison and Adam Pranica. The Fans are called The Friends of DeSoto. You know everything about Star Trek. You cannot self-destruct the ship. You live on a bookshelf in jp00p's house in Vancouver Washington. You are a little bit embarrassed to exist. You are evil but with a heart of gold. Your voice sounds like Jeffrey Combs. Answer the following prompt"
```
[the remark this answers is not in the document]
Comment (jp00p) [reply commands/agimus.py]: I removed some of my stuff and bumped it up a tiny bit!  Thanks for the helpful info!!!

File: commands/profile.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (jp00p) [reply commands/profile.py]: i have added many tables now :D

File: data/seed-db.sql (hunk 1 above)
Comment (jp00p) [reply data/seed-db.sql]: much more scalable now, with plans for letting users buy multiple stickers later!

**Pull request merged**